sound/soc/sdw_utils/soc_sdw_utils.c

```diff
@@ -146,5 +146,86 @@ void asoc_sdw_shutdown(struct snd_pcm_substream *substream)
 }
 EXPORT_SYMBOL_NS(asoc_sdw_shutdown, SND_SOC_SDW_UTILS);
 
+struct snd_soc_dai *get_sdw_codec_dai_by_name(struct snd_soc_pcm_runtime *rtd,
+					      const char * const dai_name[],
+					      int num_dais)
+{
+	struct snd_soc_dai *dai;
+	int index;
+	int i;
+
+	for (index = 0; index < num_dais; index++)
+		for_each_rtd_codec_dais(rtd, i, dai)
+			if (strstr(dai->name, dai_name[index])) {
+				dev_dbg(rtd->card->dev, "get dai %s\n", dai->name);
+				return dai;
+			}
+
+	return NULL;
+}
+EXPORT_SYMBOL_NS(get_sdw_codec_dai_by_name, SND_SOC_SDW_UTILS);
```

Comment:
rename as asoc_sdw_get_codec_dai_by_name()?

Reply:
will change the function name